x/bep3/types/params.go

```diff
@@ -0,0 +1,134 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/params"
+)
+
+// Parameter keys
+var (
+	KeyBnbDeputyAddress = []byte("BnbDeputyAddress")
+	KeyMinBlockLock     = []byte("MinBlockLock")
+	KeyMaxBlockLock     = []byte("MaxBlockLock")
+	KeySupportedAssets  = []byte("SupportedAssets")
+
+	AbsoluteMaximumBlockLock int64 = 10000
+	AbsoluteMinimumBlockLock int64 = 10
+	DefaultMinBlockLock      int64 = 20
+	DefaultMaxBlockLock      int64 = 200
+	DefaultSupportedAssets         = AssetParams{AssetParam{Denom: "kava", CoinID: "459", Limit: sdk.NewInt(1), Active: false}}
```

Comment:
```suggestion
	DefaultSupportedAssets         = AssetParams{AssetParam{Denom: "ukava", CoinID: "459", Limit: sdk.NewInt(1), Active: false}}
```

Reply:
Fixed in https://github.com/Kava-Labs/kava/pull/370/commits/ac92e26999cf32a5fae8296ba92a05b1da0cf357